homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/ball/Magic.java

```diff
@@ -0,0 +1,53 @@
+package com.tutrit.java.quickstart.homework.ball;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class Magic {
+    Logger log = LoggerFactory.getLogger("Magic Ball ");
+    Map<Integer, String> ball = new HashMap<>();
+    Random random = new Random();
```

Comment:
privat static and maybe final

Reply:
Why?

Reply:
Best practice explicitly specify the access modifier for methods and varible of class.
Veriable "random" is constant of class.

Constant class Java
A constant is a variable whose value cannot change once it has been assigned. Java doesn't have built-in support for constants, but the variable modifiers static and final can be used to effectively create one. Constants can make your program more easily read and understood by others.

Reply:
fixed